Stream_support/include/CGAL/IO/STL.h

```diff
@@ -222,6 +222,31 @@ bool read_STL(const std::string& fname,
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 // Write
 
+namespace internal {
+
+template <typename K>
+typename K::Vector_3 construct_normal_of_STL_face(const typename K::Point_3& p,
+                                                  const typename K::Point_3& q,
+                                                  const typename K::Point_3& r,
+                                                  const K& k)
+{
+  typedef typename K::FT FT;
+  typedef typename K::Vector_3 Vector;
+
+  if(k.collinear_3_object()(p, q, r))
+    return k.construct_vector_3_object()(1, 0, 0);
+
+  const Vector pq = k.construct_vector_3_object()(p, q);
+  const Vector pr = k.construct_vector_3_object()(p, r);
+  Vector res = k.construct_cross_product_vector_3_object()(pq, pr);
```

Comment:
Couldn't you just do:
`Vector res = k.construct_orthogonal_vector_3_object()(p, q, r);`